partitioner/rpc_servers/manager_pb2_grpc.py

```diff
@@ -0,0 +1,236 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
Ale ten plik chyba też zawiera nasz kod?  Jeśli tak, to przydałoby się to jakoś odseparować od części autogenerowanej.